src/Talk/Resources/config/services.yaml

```diff
@@ -0,0 +1,24 @@
+services:
+  _defaults:
+    autowire: true
+    autoconfigure: true
+    public: false
+
+  _instanceof:
+    App\Action:
+      tags: [controller.service_arguments]
+
+  App\Talk\Create\CreateTalkAction: ~
```

Comment:
Couldn't we load them all as services at once?

Reply:
The IDE doesn't detect services properly if we do that, but ok